app/views/requests/show.html.erb

```diff
@@ -1,3 +1,7 @@
+<% content_for :notifications do %>
+  <%= c 'request_notification', request: @request %>
+<% end %>
+
```

Comment:
@mattwr18 what about the request overview page? The notifications are not shown there.

Reply:
speaking of the request overview page, @mattwr18 : What about notifications at the chat page? `user_request GET    /users/:user_id/requests/:id(.:format)`
![Screenshot - 2020-09-23T112759 687](https://user-images.githubusercontent.com/38259090/93995052-d0e4e080-fd90-11ea-95e3-308ba5ccc515.png)

Reply:
> @mattwr18 what about the request overview page? The notifications are not shown there.

We talked about this, and agree that it would be cool to have, but we see that as another feature that would take more thought and time to develop. At the moment, we just have a link on the request show page that let's the editor know that there is a new message(reply) to the request and the link refreshes the page. If the user is on the index page, what should the link do? If we are polling for messages from all requests, we would need to render more info about the messages and to which request they belong. There are other cases to consider, we will create a ticket to revisit this later.

Reply:
> speaking of the request overview page, @mattwr18 : What about notifications at the chat page? `user_request GET /users/:user_id/requests/:id(.:format)`
> 

Good that you brought this up. At the moment, we are considering how helpful this page is in the first place since it sends the editor away from the request show page simply to be able to send a reply to the community member, but we could easily render an input field a the bottom of the chat conversation between an editor and that community member on the request show page itself. This would probably be better UX since we don't need to switch pages.